roles/edpm_kernel/tasks/kernelargs.yml

```diff
@@ -100,13 +100,13 @@
         - /boot/efi/EFI/fedora
 
     - name: Generate grub config
-      ansible.builtin.command: "grub2-mkconfig -o /boot/grub2/grub.cfg"
+      ansible.builtin.command: "grub2-mkconfig -o /boot/grub2/grub.cfg {{ '--update-bls-cmdline' if ansible_distribution == 'RedHat' and ansible_distribution_major_version | int >= 9 else '' }}"
```

Comment:
the defualt grub config changed to use bls in 9.3 https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/9.3_release_notes/new-features#new-features-boot-loader

but we are still generating the grub config file.

so provided --update-bls-cmdline is valid on 9.2 for adoption then we should be fine 
otherwise we need to also check the ansible_distribution_minor_version >= 3